Code/GraphMol/EnumerateStereoisomers/Wrap/rdEnumerateStereoisomers.cpp

```diff
@@ -103,6 +126,34 @@ BOOST_PYTHON_MODULE(rdEnumerateStereoisomers) {
           "Seed for random number generator.  Default=-1 means no seed.")
       .def("__setattr__", &safeSetattr);
 
+  docString = "IsomerSets structure describing how many isomers and isomer sets (batches) "
+    "will be created.  Each enhanced STEREO_OR creates two independent batches of isomers.";
+  python::class_<EnumerateStereoisomers::IsomerSets>("IsomerSets",
+		       docString.c_str())
+    .def_readwrite(
+		   "numIsomersInSet", 
+		   &EnumerateStereoisomers::IsomerSets::numIsomersInSet,
+		   "The number of isomer sets that will be generated, each isomer set"
```

Comment:
@greglandrum the isomer set definition is peppered through the docs (example here) is there a place I'm missing or are these definitions insufficient.

Reply:
I note that this one is particularly crappy,